shared-plugins/new-device-notification/new-device-notification.php

```diff
@@ -168,30 +168,32 @@ public function ip_to_city( $ip ) {
 		if ( ! function_exists( 'ip2location' ) ) {
 			$location = new stdClass();
```

Comment:
Can you use `VIP_Go_Geo_Uniques::get_country_code()` for `$location->country_short`? Also the location object is passed to the `ndn_send_email` and `ndn_notify_of_new_device` hooks, so for better compatibility it would be good to set all the same properties as [wpcom-geo-uniques](https://vip-svn.wordpress.com/plugins/wpcom-geo-uniques/wpcom-geo-uniques.php) does, e.g.:

```
$location                = new stdClass;
$location->latitude      = '43.6525';
$location->longitude     = '-79.381667';
$location->country_short = 'CA';
$location->country_long  = 'CANADA';
$location->region        = 'ONTARIO';
$location->city          = 'TORONTO';

```